### PR TITLE
doc: pyspark write example

### PR DESCRIPTION
**Pull request by wey-gu**
as titled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,11 +143,7 @@ For more information on usage, please refer to [Example](https://github.com/veso
...
 ## PySpark with Nebula Spark Connector
 
-Below is an example of calling nebula-spark-connector jar package in pyspark. For instance, we could include it in pyspark shell like:
-
-```bash
-/spark/bin/pyspark --driver-class-path nebula-spark-connector-3.0.0.jar --jars nebula-spark-connector-3.0.0.jar
-```
+Below is an example of calling nebula-spark-connector jar package in pyspark.
 
 Then, read from Nebula Graph with `metaAddress` of `"metad0:9559"` as a dataframe:
```
Comment (Nicole00) [inline README.md:148]: There's no first step, so any need with `Then`?

File: README.md [hunk 2]
```diff
@@ -175,6 +171,88 @@ You may then `show` the dataframe as follow:
...
+    "spaceName", "basketballplayer").option(
+    "label", "player").option(
+    "vidPolicy", "").option(
+    "vertexField", "_vertexId").option(
+    "batch", 1).option(
+    "metaAddress", "metad0:9559").option(
+    "graphAddress", "graphd1:9669").option(
+    "passwd", "nebula").option(
+    "writeMode", "delete").option(
+    "user", "root").save()
```
Comment (Nicole00) [inline README.md:203]: well done! great example!

File: README.md [hunk 3]
```diff
@@ -175,6 +171,88 @@ You may then `show` the dataframe as follow:
...
+  val DST_VERTEX_FIELD     = "dstVertexField"
+  val RANK_FIELD           = "rankFiled"
+  val BATCH: String        = "batch"
+  val VID_AS_PROP: String  = "vidAsProp"
+  val SRC_AS_PROP: String  = "srcAsProp"
+  val DST_AS_PROP: String  = "dstAsProp"
+  val RANK_AS_PROP: String = "rankAsProp"
+  val WRITE_MODE: String   = "writeMode"
+  val DELETE_EDGE: String  = "deleteEdge"
+```
```
Comment (Nicole00) [inline README.md:226]: should we post the option's datatype here?

File: README.md (hunk 1 above)
Comment (wey-gu) [reply README.md:148]: Will fix it :)

File: README.md (hunk 3 above)
Comment (wey-gu) [reply README.md:226]: Do you mean we should add things like bool for `deleteEdge` or? Should we expose those options here or not?

Comment (wey-gu) [reply README.md:226]: We could add comment inline on supported value/type like?

```scala
  val RANK_AS_PROP: String = "rankAsProp" # bool, i.e. true
  val WRITE_MODE: String   = "writeMode"     # string, i.e. 'update', 'insert', 'delete'
  val DELETE_EDGE: String  = "deleteEdge"    # bool, i.e. true
```

File: README.md (hunk 1 above)
Comment (wey-gu) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Nicole00) [reply README.md:232]: > We could add comment inline on supported value/type like(assuming # is a valid comment string for scala)?
> 
> ```scala
>   val RANK_AS_PROP: String = "rankAsProp" # bool, i.e. true
>   val WRITE_MODE: String   = "writeMode"  # string, i.e. 'update', 'insert', 'delete'
>   val DELETE_EDGE: String  = "deleteEdge" # bool, i.e. true
> ```

Yeah, great way. The information for options is enough now.

**Review by Nicole00 — APPROVED** [review]
LGTM

**Pull request merged**